src/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_mobile.md

```diff
@@ -32,14 +33,21 @@ Use `styles-m.less` to generate mobile-specific styles.
  ```xml
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
   <head>
+    <remove src="css/styles-l.css" />
     <css src="css/styles-m.css" />
   </head>
 </page>
 ```
 
-Media queries `@media-common`, `max screen__s` and `max` `screen__m` will be added to `styles-m.css`.
+Media queries `@media-common`, `max screen__m`, `max screen__s`, `max @screen__xs` and `max @screen__xxs`   will be added to `styles-m.css`.
 
-This will remove the desktop-specific files from your theme.
+Example of a Mixin to target screen width less than 480px:
```

Comment:
Please, add a small clarification here

```suggestion
Example of a LESS mixin to target screen width less than 480px:
```

Thank you

Reply:
@rogyar Thank you.